### ISSUE TITLE
Improving the header navbar by adding a logo and some hyperlinks like home , about us , contact page or feedback form .

### ISSUE THREAD

**Issue by usha-here** [Mohit5Upadhyay/WeatherApplication#84]
@Mohit5Upadhyay 
I would like to raise a issue . I want to improvise the navbar by adding a new logo to the page , home , about us , contact or feedback page. This would really enhance the UI design of the website.
If you could please assign me this issue , it would be really great . 
Thank you.
![Screenshot 2024-10-05 125806](https://github.com/user-attachments/assets/0512fa0b-0547-4747-898a-f77c8bd1ae8d)

**Comment by Mohit5Upadhyay** [comment]
Yes, @usha-here you can go ahead to implement these functionality

**Comment by usha-here** [comment]
@Mohit5Upadhyay can you also add the hactoberfest label and the level as it would really help me in points . thank you'

**Comment by bhaviya18** [comment]
Hello sir I would like to work on this issue, can you kindly assign this issue to me?

**Comment by Kuldeepsinghrajpoot** [comment]
I want to raise an issue with the location. When I click on the location, it takes the location from the state capital, which is not working correctly.

Let me know if this fits what you wanted!

![image](https://github.com/user-attachments/assets/e5502d07-3a07-4134-8097-dadf7916f60d)

**Comment by Akanshaku** [comment]
I would like to work on this issue pls assign me this @Mohit5Upadhyay
